src/BCF/BlockchainPolicyDeployer/Controllers/DeployerController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using RestSharp;
+using RestSharp.Authenticators;
+
+namespace BlockchainPolicyDeployer.Controllers
+{
+	[Route("blockchain_policy_deployer")]
+	[ApiController]
+	public class DeployerController : ControllerBase
+	{
+		private const int MAX_STREAM_KEY_LENGTH = 256;
+
+		public class RequestBody
+		{
+			public string json_policy { get; set; }

```

Comment:
We can use one of Newtonsoft.Json feature to ensure that when we build the json object it will have the right name format without having to modify the actual property name.

https://www.newtonsoft.com/json/help/html/JsonPropertyName.htm

```suggestion
			[JsonProperty("json_policy")]
                        public string JsonPolicy { get; set; }
```